tuf/pyca_crypto_keys.py

```diff
@@ -309,15 +309,11 @@ def create_rsa_signature(private_key, data):
   # has variable size and can be an empty string.
   if len(private_key):
     
-    # pyca/cryptography's expected exceptions when generating RSA key object:
-    # "ValueError/IndexError/TypeError:  When the given key cannot be parsed
-    # (possibly because the passphrase is wrong)."
-    # A passphrase or password is not used to 'private_key', since it should
-    # not be encrypted.
+    # Generate an RSSA-PSS signature.  Raise 'tuf.CryptoError' for any of the
```

Comment:
On lines 315-316 below, I essentially say exactly that.  Should I move those comments to line 312?